pkg/controller/configmap/configmap_controller.go

```diff
@@ -101,17 +108,37 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 		reqLogger.Info("Reconciling nsx-ncp-operator ConfigMap change")
 	} else if request.Namespace == "" && request.Name == operatortypes.NetworkCRDName {
 		reqLogger.Info("Reconciling cluster Network CRD change")
+	} else if request.Namespace == operatortypes.OperatorNamespace && request.Name == operatortypes.NcpInstallCRDName {
+		reqLogger.Info("Reconciling ncp-install CRD change")
 	} else {
 		return reconcile.Result{}, nil
 	}
 
+	// Fetch ncp-install CRD instance
+	ncpInstallCrd := &operatorv1.NcpInstall{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: operatortypes.NcpInstallCRDName,
+		Namespace: operatortypes.OperatorNamespace}, ncpInstallCrd)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Info(fmt.Sprintf("%s CRD is not found", operatortypes.NcpInstallCRDName))
+			r.status.SetDegraded(statusmanager.OperatorConfig, "NoNcpInstallCRD",
```

Comment:
`"NoNcpInstallCRD"` should be a const here https://github.com/vmware/nsx-container-plugin-operator/blob/0471eb4eb93e11570cca8a9212e4e9182c51aceb/pkg/controller/statusmanager/status_manager.go#L31

Reply:
It is in OperatorConfig status level, NoNcpInstallCRD is just the reason.

Reply:
I see, thanks Danting! :)